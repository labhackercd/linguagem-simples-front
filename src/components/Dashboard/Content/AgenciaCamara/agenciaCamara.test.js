import React from 'react';
import { render, screen } from '@testing-library/react';
import {shallow} from "enzyme/build";
import AgenciaCamaraContent from './agenciaCamara'
import NewsCard from './newsCard'
import { mount} from 'enzyme';
import MockAdapter from "axios-mock-adapter"
import axiosInstance from './../../../../auth/axiosApi'


it("should render the AgenciaCamaraContent section and match snapshot", () => {
    const component = shallow(<AgenciaCamaraContent/>);

    expect(component.exists()).toEqual(true);
    expect(component).toMatchSnapshot();
});


it("should render the NewsCard component and match snapshot ", () => {
    var mockedPropsdata={
      info: {
        retrancas: [ 'Consumidor' ],
        id: 619693,
        veiculo: 'agencia',
        ano: 2020,
        data: '2020-04-14T07:00:16-0300',
        titulo: 'Projeto de Lei yyy/2020',
        resumo: 'Resumo',
        materia: 'Texto texto texto PEC 300/2008',
        dataOrdenacao: '2020-04-14T07:00:16-0300',
        rodape: '',
        url: 'www.camara.leg.br/noticias/619693-PROJETO-DE-LEI-YYY/2020',
        temaPortal: [ 'Consumidor' ],
        temaAutomatico: [ '' ],
        comissoes: [ [Object], [Object], [Object] ],
        imagem: [ [Object], [Object] ]
      },
      sessionId: undefined
    }
    const component = shallow(<NewsCard info={mockedPropsdata} sessionId={1}/>);

    expect(component.exists()).toEqual(true);
    expect(component).toMatchSnapshot();
});

describe('Testing lifeclycle ', () => {
    var mockedResponseData = {
        "id": 1,
        "created": "2020-09-03T14:19:23.017327-03:00",
        "content_type": "news",
        "title": "Teste",
        "url": "https://www.camara.leg.br/",
        "session": 1
      }
    var mockInstance = new MockAdapter(axiosInstance);
    
    test("Test save content lifeclycle", async (done) => {
        // Return a fixed timestamp when moment().format() is called

        await mockInstance.onPost('/saved-contents/').reply(201,{mockedResponseData})
                
        const  wrapper = mount(<AgenciaCamaraContent />);
        const button = wrapper.find("#saveButton619950").at(0);
        //console.log(button.debug())
        button.simulate('click')

        done();
    });

    afterAll(() => {
        mockInstance.restore();
    });


});