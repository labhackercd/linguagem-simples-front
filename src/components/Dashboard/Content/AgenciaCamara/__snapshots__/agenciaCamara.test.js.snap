// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the AgenciaCamaraContent section 1`] = `
<div>
  <WithStyles(ForwardRef(List))
    style={
      Object {
        "maxHeight": "200px",
        "overflow": "auto",
      }
    }
  >
    <li
      key="section-619950"
    >
      <Styled(MuiBox)
        my={0.5}
      >
        <NewsCard
          info={
            Object {
              "ano": 2020,
              "data": "2020-07-30T05:49:15-0300",
              "dataOrdenacao": "2020-07-30T05:49:15-0300",
              "id": 619950,
              "materia": "No sexto debate sobre a criação de uma lei para punir a disseminação de notícias falsas na internet, especialistas criticaram o artigo 7º do projeto do Senado (), que permite às redes socais e aos serviços de mensagem requisitarem documento de identidade do usuário no caso de indícios de uso de robôs não identificados ou de contas inautênticas. Bruna Santos, da organização da sociedade civil Coding Rights, foi uma das que pediu que a Câmara dos Deputados remova o artigo 7º do texto aprovado pelos senadores. Ela defende que a identificação do usuário seja feita apenas mediante ordem judicial, como ocorre hoje. Bruna teme que o dispositivo seja utilizado para identificar ativistas de direitos humanos e grupos vulneráveis que precisam usar pseudônimos para garantir a sua liberdade de expressão. “Há ferramentas relevantes para a democracia que podem precisar ser anônimas”, ponderou. Como exemplo ela citou o Sleeping giants – coletivo de ciberativistas que combate discursos de ódio e notícias falsas, persuadindo empresas a removerem suas propagandas dos meios de comunicação que publicam desinformação. Para ela, criar mais mecanismos de identificação do usuário – além dos previstos na lei hoje – é equivocado, permitindo a instalação de sistema vigilantista no Brasil. Na visão dela, falta vontade política, e não legislação, para combater as redes organizadas de fake news existentes hoje. Sistema atual de identificação Advogado especialista em proteção de dados pessoais, Marcel Leonardi disse que a possibilidade de identificação de criminosos “já existe e funciona”. O Marco Civil da Internet prevê que provedores de aplicações na internet guardem por seis meses os dados de acesso do usuários (endereço IP, data e hora da conexão). “Esses dados podem ser cruzados com os dados guardados pelas operadoras de telecomunicações”, apontou. Essas operadoras, responsáveis pela conexão à internet, devem guardar os registros de conexão por um ano. Hoje o acesso a esses dados pelos investigadores de crimes é feito por via judicial. O procurador da República George Lodder, do grupo de apoio ao combate aos crimes cibernéticos do Ministério Público Federal (MP), concorda que os meios para investigação já estão previstos no Marco Civil da Internet. A posição do MP é de que a identificação das contas por meio da apresentação de documentos é desnecessária e fere a proteção constitucional de direitos individuais. Porém, o procurador salienta que, “o Marco Civil foi direcionado para ações de cunho civil”, não foi elaborado para resolver questões penais. Por isso, o Ministério Público sugere a inclusão na legislação de novos tipos penais para punir a disseminação de fake news. E recomenda ainda que as plataformas sejam obrigadas a comunicar o MP quando encontrarem indícios de crimes, como violação a direitos de crianças e adolescentes e racismo. Grupos organizados “Se já temos todos os elementos para coibir os crimes, por que não conseguimos avançar nessas investigações?”, perguntou a deputada “ [[Lídice da Mata]], relatora da Comissão Parlamentar Mista de Inquérito (CPMI) das Fake News. Doneda acrescentou que “o governo federal está vergonhosamente devendo à sociedade a criação da Autoridade Nacional de Proteção de Dados”. Criada pela Lei 13.853/19, o órgão ainda não foi instituído pelo governo. MP 948/20 .",
              "resumo": "",
              "retrancas": Array [
                "Ciência e Tecnologia",
              ],
              "rodape": "",
              "temaAutomatico": Array [
                "",
              ],
              "temaPortal": Array [
                "Ciência, tecnologia e comunicações",
              ],
              "titulo": "Especialistas criticam permissão para rede social pedir documento de identidade do usuário",
              "url": "www.camara.leg.br/noticias/619950-ESPECIALISTAS-CRITICAM-PERMISSAO-PARA-REDE-SOCIAL-PEDIR-DOCUMENTO-DE-IDENTIDADE-DO-USUARIO",
              "veiculo": "agencia",
            }
          }
        />
      </Styled(MuiBox)>
    </li>
    <li
      key="section-619917"
    >
      <Styled(MuiBox)
        my={0.5}
      >
        <NewsCard
          info={
            Object {
              "ano": 2020,
              "data": "2020-07-01T07:32:28-0300",
              "dataOrdenacao": "2020-07-01T07:32:28-0300",
              "deputados": Array [
                Object {
                  "nomeDeputado": "Rodrigo Maia",
                },
              ],
              "id": 619917,
              "materia": "O Plenário da Câmara dos Deputados retoma hoje a análise da PL 5131/2019, que concede uma linha de crédito especial para pequenas e médias empresas pagarem sua folha de salários em meio à crise decorrente do novo coronavírus. A MP é o item único da primeira sessão desta terça, marcada para as 13h55. Confira a pauta completa do Plenário Os deputados votarão destaques que podem modificar o projeto de lei de conversão do relator da medida provisória, deputado Zé Vitor (PL-MG). O texto foi aprovado na última quinta-feira (25). Entre outros pontos, o texto do relator autoriza os empréstimos para financiar salários e verbas trabalhistas por quatro meses. O texto original da MP previa o empréstimo para pagamento de salários por dois meses. Receita até R$ 50 milhões O relator ampliou ainda os tipos de pessoas jurídicas que poderão ter acesso ao empréstimo subsidiado. Além de empresários, sociedades empresárias e sociedades cooperativas, exceto as de crédito, poderão recorrer a ele as sociedades simples, as organizações da sociedade civil e os empregadores rurais (pessoas físicas ou jurídicas). Para pedir o empréstimo, o interessado deve ter obtido, em 2019, receita bruta anual superior a R$ 360 mil e igual ou inferior a R$ 50 milhões. O texto original fixava o limite superior em R$ 10 milhões. As operações de empréstimo poderão ocorrer até 31 de outubro de 2020, em vez de 30 de junho, como previa a MP original. Segundo o Banco Central, cerca de 107 mil empresas contrataram esse empréstimo até o dia 22 em um valor global de R$ 4 bilhões, beneficiando cerca de 1,8 milhão de trabalhadores. A estimativa do governo era atingir 12 milhões de funcionários em 1,4 milhão de empresas. Pagamento direto Se o empregador mantiver o pagamento da folha de salários na instituição financeira com a qual negociar o empréstimo, o pagamento aos funcionários deverá ser feito diretamente pela instituição. De qualquer modo, o pagamento somente poderá ser feito com depósito em conta titular do trabalhador. Além disso, o contrato deverá especificar as obrigações, entre as quais a de não demitir, sem justa causa, os empregados durante o período da contratação e por até 60 dias após a liberação da última parcela da linha de crédito. A proibição de demitir será na mesma proporção da folha de pagamento financiada. Assim, se o empregador optar por financiar o pagamento dos salários de metade dos funcionários, a proibição de demitir será restringida a esse pessoal. Além de ter de fornecer informações verdadeiras sobre sua folha de pagamento, o contratante não poderá usar os recursos para finalidade diferente do pagamento da folha ou de verbas trabalhistas. Se descumprir essas condições, o vencimento da dívida será considerado antecipado. Subsídio Chamado de Programa Emergencial de Suporte a Empregos, o mecanismo funcionará com repasse de R$ 34 bilhões da União para o Banco Nacional de Desenvolvimento Econômico e Social ( BNDES), que será o agente financeiro do governo a título gratuito, ou seja, sem remuneração. O programa prevê a participação de instituições financeiras privadas na concessão do empréstimo, que entrarão com 15% dos recursos emprestados ao tomador final. Os outros 85% virão desse valor colocado no BNDES, que repassará aos bancos e receberá os reembolsos das parcelas ou cobranças, devolvendo os recursos à União. O pedido de empréstimo poderá ser feito no valor equivalente a dois salários mínimos por empregado (R$ 2.090,00). No entanto, o relator retirou a exigência de que, para ter acesso à linha de crédito, a empresa tivesse sua folha de pagamento processada por instituição financeira participante do programa. Taxa e prazo A taxa de juros que deve ser praticada será de 3,75% ao ano, com prazo para pagar de 36 meses e carência, incluída nesse prazo, de seis meses para começar a pagar a primeira parcela. Durante a carência, os juros serão contabilizados e incorporados às parcelas. Para conceder o crédito, os bancos seguirão políticas próprias de concessão de empréstimo, podendo consultar sistemas de proteção ao crédito e registros de inadimplência referentes aos seis meses anteriores à contratação. No entanto, não poderão cobrar tarifas por saques ou pela transferência realizados pelos empregados entre sua conta salário e outras contas. Já o risco de inadimplência e eventuais perdas financeiras serão suportados na mesma proporção da participação (15% de recurso privado e 85% de recurso público). Instituições financeiras privadas e públicas estaduais participantes serão dispensadas de exigir certidões ou seguir restrições para a concessão do empréstimo, como de regularidade com o Fundo de Garantia do Tempo de Serviço (FGTS) e com a Previdência Social (Certidão Negativa de Débito – CND), regularidade eleitoral ou de pagamento do Imposto Territorial Rural (ITR). Não precisarão ainda consultar o Cadin, cadastro de restrição para contratos com o governo federal. Emendas rejeitadas Na quinta-feira passada, foram rejeitadas quatro tentativas de alterar o texto do relator: emenda rejeitada da deputada Margarida Salomão (PT-MG) estendia a linha de crédito a empresas com receita bruta acima de R$ 81 mil, contemplando pequenas empresas, com possibilidade de financiar até quatro salários mínimos por trabalhador; emenda rejeitada do deputado Alessandro Molon (PSB-RJ) previa a concessão da linha de crédito para empresas de alto crescimento com faturamento de R$ 10 milhões a R$ 100 milhões; emenda rejeitada da deputada Fernanda Melchionna (Psol-RS) estendia o programa a micro e pequenas empresas e mantinha o limite de receita bruta das empresas beneficiárias em R$ 10 milhões; destaque rejeitado, do PT, pretendia estender a linha de crédito para todas as empresas com receita bruta, em 2019, inferior a R$ 50 milhões. Saiba como funcionam as sessões virtuais do Plenário Fonte: Agência Câmara de NotíciasReportagem – Eduardo Piovesan Edição – Pierre Triboli",
              "resumo": "A MP autoriza a concessão de crédito para pequenas e médias empresas pagarem suas folhas de salários",
              "retrancas": Array [
                "Economia",
              ],
              "rodape": "Reportagem – Eduardo Piovesan
Edição – Pierre Triboli",
              "temaAutomatico": Array [
                "",
              ],
              "temaPortal": Array [
                "Economia",
              ],
              "titulo": "Câmara retoma hoje votação da MP de ajuda a empresas 2",
              "url": "www.camara.leg.br/noticias/619917-CAMARA-RETOMA-HOJE-VOTACAO-DA-MP-DE-AJUDA-A-EMPRESAS-2",
              "veiculo": "agencia",
            }
          }
        />
      </Styled(MuiBox)>
    </li>
    <li
      key="section-619903"
    >
      <Styled(MuiBox)
        my={0.5}
      >
        <NewsCard
          info={
            Object {
              "ano": 2020,
              "data": "2020-06-30T09:30:43-0300",
              "dataOrdenacao": "2020-06-30T09:30:43-0300",
              "id": 619903,
              "materia": "O Plenário da Câmara dos Deputados retoma hoje a análise da PL 5131/2019, que concede uma linha de crédito especial para pequenas e médias empresas pagarem sua folha de salários em meio à crise decorrente do novo coronavírus. A MP é o item único da primeira sessão desta terça, marcada para as 13h55. Confira a pauta completa do Plenário Os deputados votarão destaques que podem modificar o projeto de lei de conversão do relator da medida provisória, deputado Zé Vitor (PL-MG). O texto foi aprovado na última quinta-feira (25). Entre outros pontos, o texto do relator autoriza os empréstimos para financiar salários e verbas trabalhistas por quatro meses. O texto original da MP previa o empréstimo para pagamento de salários por dois meses. Receita até R$ 50 milhões O relator ampliou ainda os tipos de pessoas jurídicas que poderão ter acesso ao empréstimo subsidiado. Além de empresários, sociedades empresárias e sociedades cooperativas, exceto as de crédito, poderão recorrer a ele as sociedades simples, as organizações da sociedade civil e os empregadores rurais (pessoas físicas ou jurídicas). Para pedir o empréstimo, o interessado deve ter obtido, em 2019, receita bruta anual superior a R$ 360 mil e igual ou inferior a R$ 50 milhões. O texto original fixava o limite superior em R$ 10 milhões. As operações de empréstimo poderão ocorrer até 31 de outubro de 2020, em vez de 30 de junho, como previa a MP original. Segundo o Banco Central, cerca de 107 mil empresas contrataram esse empréstimo até o dia 22 em um valor global de R$ 4 bilhões, beneficiando cerca de 1,8 milhão de trabalhadores. A estimativa do governo era atingir 12 milhões de funcionários em 1,4 milhão de empresas. Pagamento direto Se o empregador mantiver o pagamento da folha de salários na instituição financeira com a qual negociar o empréstimo, o pagamento aos funcionários deverá ser feito diretamente pela instituição. De qualquer modo, o pagamento somente poderá ser feito com depósito em conta titular do trabalhador. Além disso, o contrato deverá especificar as obrigações, entre as quais a de não demitir, sem justa causa, os empregados durante o período da contratação e por até 60 dias após a liberação da última parcela da linha de crédito. A proibição de demitir será na mesma proporção da folha de pagamento financiada. Assim, se o empregador optar por financiar o pagamento dos salários de metade dos funcionários, a proibição de demitir será restringida a esse pessoal. Além de ter de fornecer informações verdadeiras sobre sua folha de pagamento, o contratante não poderá usar os recursos para finalidade diferente do pagamento da folha ou de verbas trabalhistas. Se descumprir essas condições, o vencimento da dívida será considerado antecipado. Subsídio Chamado de Programa Emergencial de Suporte a Empregos, o mecanismo funcionará com repasse de R$ 34 bilhões da União para o Banco Nacional de Desenvolvimento Econômico e Social ( BNDES), que será o agente financeiro do governo a título gratuito, ou seja, sem remuneração. O programa prevê a participação de instituições financeiras privadas na concessão do empréstimo, que entrarão com 15% dos recursos emprestados ao tomador final. Os outros 85% virão desse valor colocado no BNDES, que repassará aos bancos e receberá os reembolsos das parcelas ou cobranças, devolvendo os recursos à União. O pedido de empréstimo poderá ser feito no valor equivalente a dois salários mínimos por empregado (R$ 2.090,00). No entanto, o relator retirou a exigência de que, para ter acesso à linha de crédito, a empresa tivesse sua folha de pagamento processada por instituição financeira participante do programa. Taxa e prazo A taxa de juros que deve ser praticada será de 3,75% ao ano, com prazo para pagar de 36 meses e carência, incluída nesse prazo, de seis meses para começar a pagar a primeira parcela. Durante a carência, os juros serão contabilizados e incorporados às parcelas. Para conceder o crédito, os bancos seguirão políticas próprias de concessão de empréstimo, podendo consultar sistemas de proteção ao crédito e registros de inadimplência referentes aos seis meses anteriores à contratação. No entanto, não poderão cobrar tarifas por saques ou pela transferência realizados pelos empregados entre sua conta salário e outras contas. Já o risco de inadimplência e eventuais perdas financeiras serão suportados na mesma proporção da participação (15% de recurso privado e 85% de recurso público). Instituições financeiras privadas e públicas estaduais participantes serão dispensadas de exigir certidões ou seguir restrições para a concessão do empréstimo, como de regularidade com o Fundo de Garantia do Tempo de Serviço (FGTS) e com a Previdência Social (Certidão Negativa de Débito – CND), regularidade eleitoral ou de pagamento do Imposto Territorial Rural (ITR). Não precisarão ainda consultar o Cadin, cadastro de restrição para contratos com o governo federal. Emendas rejeitadas Na quinta-feira passada, foram rejeitadas quatro tentativas de alterar o texto do relator: emenda rejeitada da deputada Margarida Salomão (PT-MG) estendia a linha de crédito a empresas com receita bruta acima de R$ 81 mil, contemplando pequenas empresas, com possibilidade de financiar até quatro salários mínimos por trabalhador; emenda rejeitada do deputado Alessandro Molon (PSB-RJ) previa a concessão da linha de crédito para empresas de alto crescimento com faturamento de R$ 10 milhões a R$ 100 milhões; emenda rejeitada da deputada Fernanda Melchionna (Psol-RS) estendia o programa a micro e pequenas empresas e mantinha o limite de receita bruta das empresas beneficiárias em R$ 10 milhões; destaque rejeitado, do PT, pretendia estender a linha de crédito para todas as empresas com receita bruta, em 2019, inferior a R$ 50 milhões. Saiba como funcionam as sessões virtuais do Plenário Fonte: Agência Câmara de NotíciasReportagem – Eduardo Piovesan Edição – Pierre Triboli",
              "resumo": "A MP autoriza a concessão de crédito para pequenas e médias empresas pagarem suas folhas de salários",
              "retrancas": Array [
                "Economia",
              ],
              "rodape": "Reportagem – Eduardo Piovesan
Edição – Pierre Triboli",
              "temaAutomatico": Array [
                "",
              ],
              "temaPortal": Array [
                "Economia",
              ],
              "titulo": "Câmara retoma hoje votação da MP de ajuda a empresas",
              "url": "www.camara.leg.br/noticias/619903-CAMARA-RETOMA-HOJE-VOTACAO-DA-MP-DE-AJUDA-A-EMPRESAS",
              "veiculo": "agencia",
            }
          }
        />
      </Styled(MuiBox)>
    </li>
    <li
      key="section-617681"
    >
      <Styled(MuiBox)
        my={0.5}
      >
        <NewsCard
          info={
            Object {
              "ano": 2020,
              "data": "2020-06-30T08:12:08-0300",
              "dataOrdenacao": "2020-06-30T08:12:08-0300",
              "id": 617681,
              "materia": "O Projeto de Lei 5131/19 propõe alterações em diversas leis para combater o garimpo ilegal de ouro no Brasil. Entre outros pontos, o texto determina que o transporte de ouro deverá ser sempre acompanhado de nota fiscal eletrônica e de guia de transporte expedida digitalmente pelo detentor da permissão da lavra (conhecida pela sigla PLG). A PEC 6/2019, alguma coisa...A proposta tramita na Câmara dos Deputados. Conforme o texto, a guia de transporte deverá informar a massa do ouro bruto transportado e dados sobre sua origem e destino. O documento será emitido por um sistema implantado e operado pela Agência Nacional de Mineração (ANM), e terá validade até a consumação da venda do mineral. Além da emissão da guia, o sistema armazenará dados sobre a produção, o fluxo, a venda e o transporte de ouro, além das notas fiscais eletrônicas das vendas do mineral. Atualmente, a Lei 12.844/13 determina que o transporte de ouro deve ser acompanhado apenas da nota fiscal tradicional e da cópia do título autorizativo de lavra, \\"não se exigindo outro documento\\". Controle O deputado Camilo Capiberibe, autor da proposta, defende que o maior controle do transporte e comercialização do ouro extraído pode potencializar as ações de combate ao ouro ilegal. “A dificuldade na comercialização do produto de origem ilícita pode, com menores custos e impactos sociais, promover a legalização das áreas de extração, cujos proprietários, atualmente, não veem motivos para arcar com os custos dessa regularização”, disse. Ele afirma ainda que o projeto busca criar um aparato fiscalizatório semelhante ao que existe para o transporte e a comercialização de madeira nativa. Novas funções Pela proposta, a ANM terá competência para fiscalizar o transporte de minérios e compartilhar as informações do setor, fornecidas pelos titulares de direitos minerários, com outros agentes fiscalizadores, para auxiliar o combate às lavras ilegais. O projeto altera ainda a Lei dos Crimes Ambientais para aumentar as penas para crimes relacionados à mineração. A proposta prevê punição - detenção, de seis meses a um ano, e multa - para que receber ou adquirir mineral para fins comerciais ou industriais sem exigir a guia de transporte do vendedor. O texto também regulamenta a multa diária prevista na lei para pesquisa, lavra ou extração de recursos minerais sem autorização. Ela não poderá ser inferior a um nem superior a dez salários mínimos. A multa é definida pelo juiz. Tramitação A proposta será analisada pelas comissões de Minas e Energia; Finanças e Tributação; e Constituição e Justiça e de Cidadania. Depois seguirá para o Plenário da Câmara.Reportagem – Janary Júnior",
              "resumo": "",
              "retrancas": Array [
                "Administração Pública",
              ],
              "rodape": "Reportagem – Janary Júnior",
              "temaAutomatico": Array [
                "",
              ],
              "temaPortal": Array [
                "Política e administração pública",
              ],
              "titulo": "Projeto institui nota fiscal e guia eletrônicas para venda e transporte de ouro",
              "url": "www.camara.leg.br/noticias/617681-PROJETO-INSTITUI-NOTA-FISCAL-E-GUIA-ELETRONICAS-PARA-VENDA-E-TRANSPORTE-DE-OURO",
              "veiculo": "agencia",
            }
          }
        />
      </Styled(MuiBox)>
    </li>
    <li
      key="section-619881"
    >
      <Styled(MuiBox)
        my={0.5}
      >
        <NewsCard
          info={
            Object {
              "ano": 2020,
              "data": "2020-06-15T08:51:27-0300",
              "dataOrdenacao": "2020-06-15T08:51:27-0300",
              "id": 619881,
              "materia": "Teste",
              "resumo": "",
              "retrancas": Array [
                "Administração Pública",
              ],
              "rodape": "",
              "temaAutomatico": Array [
                "",
              ],
              "temaPortal": Array [
                "Política e administração pública",
              ],
              "titulo": "Ultimas 1",
              "url": "www.camara.leg.br/noticias/619881-ULTIMAS-1",
              "veiculo": "agencia",
            }
          }
        />
      </Styled(MuiBox)>
    </li>
    <li
      key="section-619742"
    >
      <Styled(MuiBox)
        my={0.5}
      >
        <NewsCard
          info={
            Object {
              "ano": 2020,
              "comissoes": Array [
                Object {
                  "apelido": "COMISSÃO DE DESENVOLVIMENTO URBANO",
                  "id": 2006,
                  "sigla": "CDU",
                },
                Object {
                  "apelido": "COMISSÃO DE TRABALHO, ADMINISTRAÇÃO E SERVIÇO PÚBLICO",
                  "id": 2015,
                  "sigla": "CTASP",
                },
              ],
              "data": "2020-05-28T09:36:20-0300",
              "dataOrdenacao": "2020-05-28T09:36:20-0300",
              "id": 619742,
              "materia": "sdf",
              "resumo": "",
              "retrancas": Array [
                "Administração Pública",
              ],
              "rodape": "",
              "temaAutomatico": Array [
                "",
              ],
              "temaPortal": Array [
                "Política e administração pública",
              ],
              "titulo": "Notícia urgente 2",
              "url": "www.camara.leg.br/noticias/619742-NOTICIA-URGENTE-2",
              "veiculo": "agencia",
            }
          }
        />
      </Styled(MuiBox)>
    </li>
    <li
      key="section-619739"
    >
      <Styled(MuiBox)
        my={0.5}
      >
        <NewsCard
          info={
            Object {
              "ano": 2020,
              "comissoes": Array [
                Object {
                  "apelido": "COMISSÃO DE CONSTITUIÇÃO E JUSTIÇA E DE CIDADANIA",
                  "id": 2003,
                  "sigla": "CCJC",
                },
                Object {
                  "apelido": "COMISSÃO DE TRABALHO, ADMINISTRAÇÃO E SERVIÇO PÚBLICO",
                  "id": 2015,
                  "sigla": "CTASP",
                },
              ],
              "data": "2020-04-30T13:24:08-0300",
              "dataOrdenacao": "2020-04-30T13:24:08-0300",
              "id": 619739,
              "materia": "sdf",
              "resumo": "",
              "retrancas": Array [
                "Administração Pública",
              ],
              "rodape": "",
              "temaAutomatico": Array [
                "",
              ],
              "temaPortal": Array [
                "Política e administração pública",
              ],
              "titulo": "Notícia urgente com comissoes -",
              "url": "www.camara.leg.br/noticias/619739-NOTICIA-URGENTE-COM-COMISSOES--",
              "veiculo": "agencia",
            }
          }
        />
      </Styled(MuiBox)>
    </li>
    <li
      key="section-619736"
    >
      <Styled(MuiBox)
        my={0.5}
      >
        <NewsCard
          info={
            Object {
              "ano": 2020,
              "data": "2020-04-29T07:20:42-0300",
              "dataOrdenacao": "2020-04-29T07:20:42-0300",
              "id": 619736,
              "materia": "sdf",
              "resumo": "",
              "retrancas": Array [
                "Administração Pública",
              ],
              "rodape": "",
              "temaAutomatico": Array [
                "",
              ],
              "temaPortal": Array [
                "Política e administração pública",
              ],
              "titulo": "Notícia urgente",
              "url": "www.camara.leg.br/noticias/619736-NOTICIA-URGENTE",
              "veiculo": "agencia",
            }
          }
        />
      </Styled(MuiBox)>
    </li>
    <li
      key="section-619696"
    >
      <Styled(MuiBox)
        my={0.5}
      >
        <NewsCard
          info={
            Object {
              "ano": 2020,
              "data": "2020-04-14T07:04:06-0300",
              "dataOrdenacao": "2020-04-14T07:04:06-0300",
              "id": 619696,
              "materia": "PEC 300/2008 será votada hoje no Plenário",
              "resumo": "",
              "retrancas": Array [
                "Consumidor",
              ],
              "rodape": "",
              "temaAutomatico": Array [
                "",
              ],
              "temaPortal": Array [
                "Consumidor",
              ],
              "titulo": "PEC 300/2008 será votada hoje no Plenário",
              "url": "www.camara.leg.br/noticias/619696-PEC-300/2008-SERA-VOTADA-HOJE-NO-PLENARIO",
              "veiculo": "agencia",
            }
          }
        />
      </Styled(MuiBox)>
    </li>
    <li
      key="section-619693"
    >
      <Styled(MuiBox)
        my={0.5}
      >
        <NewsCard
          info={
            Object {
              "ano": 2020,
              "comissoes": Array [
                Object {
                  "apelido": "COMISSÃO DE CONSTITUIÇÃO E JUSTIÇA E DE CIDADANIA",
                  "id": 2003,
                  "sigla": "CCJC",
                },
                Object {
                  "apelido": "COMISSÃO DE DEFESA DO CONSUMIDOR",
                  "id": 2004,
                  "sigla": "CDC",
                },
                Object {
                  "apelido": "COMISSÃO DE CULTURA",
                  "id": 536996,
                  "sigla": "CCULT",
                },
              ],
              "data": "2020-04-14T07:00:16-0300",
              "dataOrdenacao": "2020-04-14T07:00:16-0300",
              "id": 619693,
              "imagem": Array [
                Object {
                  "id": 619895,
                  "nome": "alunos_rurais",
                  "sizes": Object {
                    "large": Object {
                      "height": "684",
                      "url": "https://socithom.camara.gov.br/portal/portal-backend-wordpress/wp-content/uploads/midias/image/2020/06/alunos-rurais-1024x684.jpg",
                      "width": "1024",
                    },
                    "medium": Object {
                      "height": "200",
                      "url": "https://socithom.camara.gov.br/portal/portal-backend-wordpress/wp-content/uploads/midias/image/2020/06/alunos-rurais-300x200.jpg",
                      "width": "300",
                    },
                    "medium_large": Object {
                      "height": "513",
                      "url": "https://socithom.camara.gov.br/portal/portal-backend-wordpress/wp-content/uploads/midias/image/2020/06/alunos-rurais-768x513.jpg",
                      "width": "768",
                    },
                    "post-thumbnail": Object {
                      "height": "1047",
                      "url": "https://socithom.camara.gov.br/portal/portal-backend-wordpress/wp-content/uploads/midias/image/2020/06/alunos-rurais-1568x1047.jpg",
                      "width": "1568",
                    },
                    "thumbnail": Object {
                      "height": "150",
                      "url": "https://socithom.camara.gov.br/portal/portal-backend-wordpress/wp-content/uploads/midias/image/2020/06/alunos-rurais-150x150.jpg",
                      "width": "150",
                    },
                  },
                  "url": "https://socithom.camara.gov.br/portal/portal-backend-wordpress/wp-content/uploads/midias/image/2020/06/alunos-rurais.jpg",
                },
                Object {
                  "id": 619896,
                  "nome": "Presidência",
                  "sizes": Object {
                    "large": Object {
                      "height": "683",
                      "url": "https://socithom.camara.gov.br/portal/portal-backend-wordpress/wp-content/uploads/midias/image/2020/06/img20200622134709243-1024x683.jpg",
                      "width": "1024",
                    },
                    "medium": Object {
                      "height": "200",
                      "url": "https://socithom.camara.gov.br/portal/portal-backend-wordpress/wp-content/uploads/midias/image/2020/06/img20200622134709243-300x200.jpg",
                      "width": "300",
                    },
                    "medium_large": Object {
                      "height": "512",
                      "url": "https://socithom.camara.gov.br/portal/portal-backend-wordpress/wp-content/uploads/midias/image/2020/06/img20200622134709243-768x512.jpg",
                      "width": "768",
                    },
                    "post-thumbnail": Object {
                      "height": "1045",
                      "url": "https://socithom.camara.gov.br/portal/portal-backend-wordpress/wp-content/uploads/midias/image/2020/06/img20200622134709243-1568x1045.jpg",
                      "width": "1568",
                    },
                    "thumbnail": Object {
                      "height": "150",
                      "url": "https://socithom.camara.gov.br/portal/portal-backend-wordpress/wp-content/uploads/midias/image/2020/06/img20200622134709243-150x150.jpg",
                      "width": "150",
                    },
                  },
                  "url": "https://socithom.camara.gov.br/portal/portal-backend-wordpress/wp-content/uploads/midias/image/2020/06/img20200622134709243.jpg",
                },
              ],
              "materia": "Texto texto texto PEC 300/2008",
              "resumo": "Resumo",
              "retrancas": Array [
                "Consumidor",
              ],
              "rodape": "",
              "temaAutomatico": Array [
                "",
              ],
              "temaPortal": Array [
                "Consumidor",
              ],
              "titulo": "Projeto de Lei yyy/2020",
              "url": "www.camara.leg.br/noticias/619693-PROJETO-DE-LEI-YYY/2020",
              "veiculo": "agencia",
            }
          }
        />
      </Styled(MuiBox)>
    </li>
  </WithStyles(ForwardRef(List))>
</div>
`;
